### DISCUSSION TITLE
Are the friction and damping parameters of the joints actually used in the calculation of the dynamics?

### DISCUSSION THREAD

**Discussion by zixinz990** [stack-of-tasks/pinocchio#2285] [Q&A] [answered]
I noticed that the model includes membership variables for friction and damping, but in using Pinocchio, I felt that these variables are not used in the joint modeling and calculation of the dynamics.

**Comment by jcarpent** [comment, marked as answer]
Starting from Pinocchio 3, this is handled by default.
